server/README.md

```diff
@@ -395,3 +395,42 @@ curl --location --request POST 'http://localhost:3333/api/v1/event' \
     "status": "OK"
 }
 ```
+
+
+## Send an event sequence
+
+### Example cURL Command:
+
+```
+curl --location --request POST 'http://localhost:3333/api/v1/sequence' \
+--header 'Content-Type: application/json' \
+--data-raw '[{
```

Comment:
Just a question, really: Do I really post "just" the array, or do I post an object with some well-known key (seqevent?) whose value is the array?